### DISCUSSION TITLE
How can I disable dangerjs from touching Bitbucket Cloud's pipeline status?

### DISCUSSION THREAD

**Discussion by binary64** [danger/danger-js#1146] [Q&A] [answered]
I see this on my PR:
![image](https://user-images.githubusercontent.com/1680627/124369552-ac455f80-dc64-11eb-93b9-f61f41b3561d.png)

How can I disable this? It's overwriting my Bitbucket Pipeline status.

I just want dangerjs to return a non-0 exit code when there is a warning or failure.

Many thanks!

**Comment by orta** [comment, marked as answer]
Perhaps `--no-publish-check` disables that in bitbucket, if not, I'd accept a PR which makes it work
